### PR TITLE
Upgrade sentry logger in order to support release tracking

### PR DESCRIPTION
**Pull request by OskarKjellin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -48,17 +48,17 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>1.7.5</version>
+      <version>1.7.22</version>
```
Comment (rydenius) [inline pom.xml]: Why not go for latest 1.7.24 while you are at it?

File: pom.xml [hunk 2]
```diff
@@ -48,17 +48,17 @@
...
     </dependency>
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
-      <version>1.0.13</version>
+      <version>1.1.9</version>
```
Comment (rydenius) [inline pom.xml]: Why not go for latest 1.2.1 while you are at it (or at least 1.1.10 which is the latest on the 1.1.x branch)?

**Comment by OskarKjellin** [comment]
@rydenius I'll update the dependencies. They weren't released when I made the PR

**Comment by mattnworb** [comment]
just curious, are the bumps to slf4j and logback versions because raven-logback needs something >= those versions?

**Comment by mattnworb** [comment]
LGTM in either case

**Comment by OskarKjellin** [comment]
Yes @mattnworb

**Comment by OskarKjellin** [comment]
@rydenius Updated deps

**Comment by mattnworb** [comment]
👍

**Pull request merged**